app/reducers/welcome.js

```diff
@@ -27,6 +34,28 @@ export default (state = INITIAL_STATE, action) => {
         ...state,
         fadeIn: !state.fadeIn
       }
+    case ACTIONS.SAVE_SETTINGS:
```

Comment:
What about using settings reducer instead of code duplication?

Reply:
So I was reviewing this PR, and cloned this branch and did some testing manually, I agree with @abahmed about using just the reducer that the `Settings component` has in order to avoid duplication of code and prevent a potential bug, with the current implementation you have the `ACTIONS.SAVE_SETTINGS` reducer is being dispatched twice

Reply:
Also we have another option instead of reusing the reducer for `Settings component` we could create another reducer that could be called like `case ACTIONS.SAVE_LANGUAGE_SETTINGS:`
because at least in the welcome component we probably won't let the user configure more settings than `language`.

Thoughts?

Reply:
Thanks @abahmed  @mjimenez07 for the reviews, i already done the changes suggested. Having other reducer should be discuss as a further improvement taking in account that in the future the user could be asked some other settings like email for backups or the background color etc.